modules/ipi-preparing-reinstall-cluster-bare-metal.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare-metal
```

Comment:
is this file related to the context of this PR ?

Reply:
I've removed the file. It was included by accident.